src/tests/App.test.js

```diff
@@ -1,5 +1,5 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import App from "../containers/App"
```

Comment:
Great find 💯 

Thanks for fixing this and contributing on our project. We would be delighted to see your contributions again in the future!